src/index.css

```diff
@@ -1,27 +1,264 @@
-body {
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@200;300;400&display=swap');
+
+* {
+  padding: 0;
   margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+  box-sizing: border-box;
```

Comment:
- [ ] When it comes to styling. Please make sure your styling should match the given design. I will share screenshots of your app with marked errors:

<img width="1439" alt="Screenshot 2023-05-14 at 00 26 58" src="https://github.com/Oscah01/the-book-store/assets/69011963/e410eac0-ea78-4191-b543-d21f404b0ee8">

<img width="1439" alt="Screenshot 2023-05-14 at 00 25 53" src="https://github.com/Oscah01/the-book-store/assets/69011963/ec8db62b-e511-41ed-aff8-2373af7387e3">